utils/utils.go

```diff
@@ -27,3 +30,8 @@ func PrettyPrintArray(value interface{}) string {
 	}
 	return string(b)
 }
+
+func RemoveRegexp(value string, expression string) string {
```

Comment:
See my other comments, but I think we should not use this. It's too specific to one installation and one language. Most of digitalstrom users are german speaking. So by just removing `light` it doesn't help for 90% of the users. Same for me, I use `light` but with an underscore and not a space.

Reply:
I just reimplemented it as an optional parameter that you can configure and would work for other types of suffixes.

Reply:
good for me